### PR TITLE
Create the resource 'pdns_authoritative_install'

### PR DESCRIPTION
**Pull request by jmauro**
Hello,

The idea is to create the 'pdns_authoritative_install' the same way we have 'pdns_recursor_install'.

cc: @criteo-forks/sre-core-infra

### CODE REVIEW

**Review by olive42 — APPROVED** [review]

File: metadata.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name             'pdns'
-maintainer       'Aetrion, LLC DBA DNSimple'
+maintainer       'DNSimple Corp'
```
Comment (Annih) [inline metadata.rb:2]: Why are you changing that?

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2018 DNSimple Corp.
```
Comment (Annih) [inline LICENSE:189]: Why are you changing that?

File: resources/pdns_authoritative_install.rb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  end
+
+end
+
+action :uninstall do
+  package pkg_name do
+    action :remove
+  end
+
+  package pkg_name_debug do
```
Comment (Annih) [inline resources/pdns_authoritative_install.rb:59]: Why are you installing debugs in the uninstall action?

File: resources/pdns_authoritative_install.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    only_if { new_resource.debug }
+  end
+
+  copy_properties_to(pdns_recursor_repo(new_resource.instance_name) { action :delete })
```
Comment (Annih) [inline resources/pdns_authoritative_install.rb]: You meant `pdns_authoritative_repo`

File: resources/pdns_authoritative_repo_debian.rb [hunk 5]
```diff
@@ -17,53 +17,48 @@
...
+resource_name :pdns_authoritative_repo_debian
 
-provides :pdns_authoritative_install, platform: 'ubuntu' do |node|
-  node['platform_version'].to_f >= 14.04
-end
-
-provides :pdns_authoritative_install, platform: 'debian' do |node|
-  node['platform_version'].to_i >= 8
-end
+provides :pdns_authoritative_repo, platform_family: 'debian'
```
Comment (Annih) [inline resources/pdns_authoritative_repo_debian.rb]: Are you sure you are compatible with `Debian < 8` & `Ubuntu < 14.04` ?

**Review by Annih — CHANGES_REQUESTED** [review]
You should be more explicit on the fact you are doing a breaking change by renaming the install resource into a repo one.

File: resources/pdns_authoritative_install.rb (hunk 3 above)
Comment (jmauro) [reply resources/pdns_authoritative_install.rb:59]: Fix

File: resources/pdns_authoritative_install.rb (hunk 4 above)
Comment (jmauro) [reply resources/pdns_authoritative_install.rb]: fix

File: resources/pdns_authoritative_repo_debian.rb (hunk 5 above)
Comment (jmauro) [reply resources/pdns_authoritative_repo_debian.rb]: The support was clearly stated in the metadata.

**Review by Annih — APPROVED** [review]

**Pull request merged**